majority-multisign/src/MajorityMultiSign/Contracts.hs

```diff
@@ -67,19 +67,24 @@ import Plutus.Contract (
  )
 import Plutus.Contract.Types (mapError)
 import Plutus.Contracts.Currency (CurrencyError (CurContractError), currencySymbol, mintContract)
+import Plutus.V1.Ledger.Ada (lovelaceValueOf)
 import Plutus.V1.Ledger.Api (
   Datum (Datum, getDatum),
   Redeemer (Redeemer),
   ToData,
   fromBuiltinData,
  )
-import Plutus.V1.Ledger.Value (assetClass, assetClassValue, assetClassValueOf)
+import Plutus.V1.Ledger.Value (Value, assetClass, assetClassValue, assetClassValueOf)
 import PlutusTx (toBuiltinData)
 import PlutusTx.List.Natural qualified as Natural
 import PlutusTx.Natural (Natural)
 import PlutusTx.Numeric.Extra ((^-))
 import PlutusTx.Prelude hiding (foldMap, (<>))
 
+-- | Add min lovelace to a value
+addMinLovelace :: Value -> Value
```

Comment:
You might want to export this function and use it in integration tests. But, are you quite sure it's necessary in production? Will CardStarter have to spend 2 Ada every month on the transfer?

Reply:
Keep in mind that 2ada is like $2, but possibly, it'll depend on how many inputs vs how many outputs there are, since each input will have about 2ada in it.
Also, these min lovelace values get added by bot-plutus-interface if they're not already there, so this is just moving that logic a little more to the forefront so the tests pass.